quick-start/examples/n7.min.js

```diff
@@ -0,0 +1,29 @@
+import { Scene, Circle, Rectangle, Color }
+    from "https://unpkg.com/pencil.js/dist/pencil.esm.js";
+
+const scene = new Scene(undefined, {
+    fill: new Color(), // Use the Color object as fill option
+});
+const { width, height } = scene;
+scene.cursorPosition.set(scene.center).subtract(0, height / 4);
+
+const sun = new Circle(scene.center.add(0, height / 2), 100, {
+    fill: "orange",
+});
+
+const ocean = new Rectangle([0, height / 2], width, height / 2, {
+    fill: "#28488f",
+});
+
+scene
+    .add(sun, ocean)
+    .startLoop()
+    .on("draw", () => {
+        sun.position.lerp(scene.cursorPosition, 0.05);
+
+        // Ratio of the sun setting
+        const setting = (sun.position.y - (height / 4)) / (height / 2);
```

Comment:
I must have skipped this line and further down got confused as to what `setting` (meaning "option") you were storing in there.
`sunSettingProgress` or `sunSettingRatio` may be longer, but are more descriptive names.
If this is a percent value from 0 (full day light) to 100 (full sunset), consider `sunSettingPercentage`.